transform/mattermost-analytics/models/staging/mattermost2/_mattermost2__sources.yml

```diff
@@ -44,6 +49,7 @@ sources:
       - name: config_privacy
       - name: config_rate
       - name: config_rupport
```

Comment:
I wonder if an easier thing to do would be to just drop the table and remove this field (and the freshness tag). I assume it is just a misspell of `config_support`.

Reply:
I agree with this approach. This is what I meant by `It already helped identify some source tables that can be deprecated.`. Most of these tables have < 1k rows.

The only exception is `mattermost2.event_mobile`, which contains ~ 56 million rows, but with maximum `received_at` date equal to 2020-03-06. Since we're considering telemetry after 2020-04-01, this table can be archived. A nice side-effect is a small gain in performance.